client/actions/loginActions.js

```diff
@@ -55,9 +47,7 @@ export function getUserByAccessId(body) {
             .then((response) => {
                 dispatch(getUserByAccessIdAction(response))
             })
-            .catch((err) => {
-                console.log("error==========>>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}